### DISCUSSION TITLE
增加更多的微操权限

### DISCUSSION THREAD

**Discussion by RuoYear** [XiaoMi/ha_xiaomi_home#406] [Ideas]
官方能接入HA非常好，终于起码比单纯在米家里设置来的更为方便。

所以在此提个建议，例如我有部闲置多年的设备：	
小米有品智能窗帘电机（Wi-Fi版）lumi.curtain.hagl05，多年前一直发生遇阻不停的情况导致已闲置很久；
找客服的话是建议我换台新的，但换新的话怎么对得起我们的动手能力嘛；
但米家APP可没这么“专业”的操作。

所以也请增加一些可调节的空间，例如几年前就建议过的“可手动设定窗帘最大值和窗帘最小值”的阈值，方便自行定义开关范围；
并且能跟小爱音箱更好联动而不会导致总是理解错指令。

**Comment by JimmyKmi** [comment]
这是设备厂商的自动化提供问题，需要联系厂商解决
